app/main.py

```diff
@@ -1,8 +1,40 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str) \
+            -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: int,
+                 clean_power: int, average_rating: int,
+                 count_of_ratings: int) -> None:
+        self.distance_from_the_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list) -> float:
+        income = 0
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                income += self.calculate_washing_price(car)
+                self.wash_single_car(car)
+        return round(income, 1)
+
+    def wash_single_car(self, car: Car) -> None:
+        car.clean_mark = self.clean_power
+
+    def calculate_washing_price(self, car: Car) -> float:
+        single_car_wash_price = (
+            car.comfort_class * (self.clean_power - car.clean_mark)
+            * (self.average_rating / self.distance_from_the_city_center))
+        return round(single_car_wash_price, 1)
+
+    def rate_service(self, rating: int) -> None:
+        self.average_rating = (round
+                               ((self.average_rating
+                                 * self.count_of_ratings + rating)
+                                / (self.count_of_ratings + 1), 1))
```

Comment:
Code Style issue: Similar to the previous comment, this mathematical expression is quite complex and could be broken down into multiple steps or grouped with parentheses to improve readability.